### ISSUE TITLE
[BUG] Tabbed browsing: complete different within the same browser but different OS

### ISSUE THREAD

**Issue by math-GH** [FreshRSS/FreshRSS#4210]
That is very strange.

I use 1.20.0-dev on my online instance and use it with different devices.

Today I checked the tabbed browsing.

Firefox 96.0.3 on Windows 10: `37 tab steps`
![grafik](https://user-images.githubusercontent.com/1645099/152829275-c9855c8d-5073-42e0-9299-f04e9903a309.png)

Firefox 96.0.3 on MacOS 11.6: `only 4 tab steps`
![grafik](https://user-images.githubusercontent.com/1645099/152829475-f944db26-9b8e-472a-85c1-1b75e0b118de.png)

You see that there is a huge difference between both Operation systems in tabbing controls.

**To Reproduce**
1. Enable `Show tabbing order` in `Accessibility` tab in your Firefox browser dev bar

**Expected behavior**
Same tabbing behavior

**Comment by Frenzie** [comment]
From https://github.com/FreshRSS/FreshRSS/pull/4205 ? But I'm not sure if it looking different is necessarily a bug if it simply looks how it's supposed to look in the OS. :)

**Comment by math-GH** [comment]
It is the same instance at the same git commit at the same browser. Just different operating system.

**Comment by Frenzie** [comment]
Is this the bug? https://bugzilla.mozilla.org/show_bug.cgi?id=759922

Edit: btw, Firefox on Linux is the same as on Windows in this regard.

**Comment by math-GH** [comment]
it is not this bug.

I checked on Mac also the Safari browser. It has also a very very small set of tabs. Maybe Mac does not like tab browsing? Somewhere I found an article, that in Safari you cannot select radio buttons and buttons with tab.....

**Comment by math-GH** [comment]
I close it. I guess we cannot change/fix it. And I do not have a Mac anymore so I cannot test it anymore
